File: java/hello/GuessNumber.java
import java.util.Random;
import java.util.Scanner;

public class GuessNumber {
    private static final int MAX = 3;
    public static void main(final String... args) {
        Scanner s = new Scanner(System.in);
        int number = new Random().nextInt(MAX) + 1;
        System.out.println(String.format("Guess my number (it is between 1 and %d)", MAX));
        while (true) {
            int guess = s.nextInt();
            System.out.println(guess == number ?
                "You guessed correctly!" :
                "Haha, wrong!");
        }
    }
}

File: kotlin/hello/GuessNumber.kt
import java.util.Random
import java.util.Scanner


const val MAX = 3
fun main() {
    val s = Scanner(System.`in`)
    val number = Random().nextInt(MAX) + 1
    println("Guess my number (it is between 1 and $MAX)")
    while (true) {
        val guess = s.nextInt()
        println(if (guess == number)
            "You guessed correctly!" else
            "Haha, wrong!")
    }
}
